packages/wrangler/README.md

```diff
@@ -14,73 +14,55 @@
 
 ## Quick Start
 
+To get started quickly with a new project, run the command below and follow the interactive instructions:
+
 ```bash
-# Make a javascript file
-echo "export default { fetch() { return new Response('hello world') } }" > index.js
-# try it out
-npx wrangler dev index.js
-# and then deploy it
-npx wrangler deploy index.js --name my-worker
-# visit https://my-worker.<your workers subdomain>.workers.dev
+npx wrangler init my-worker
```

Comment:
the instructions in https://developers.cloudflare.com/workers/get-started/guide/ use `npm create cloudflare@latest -- my-first-worker` rather than `wrangler init`. i'd suggest using the same command here as in the guide. is there a reason you went with `init`?

Reply:
I was leaning towards `wrangler init` because this is the README specifically for the Wrangler package, and it felt like introducing C3 at that stage might be overcomplicating? Happy to change though